quickcomm/views.py

```diff
@@ -61,6 +62,55 @@ def login(request):
         form = CreateLoginForm()
     return render(request, 'quickcomm/login.html', {'form': form})
 
+@login_required
+def post_view(request, post_id):
+    objects = Post.objects.all()
+    post = objects.get(pk=post_id)
+    post_comments = Comment.objects.filter(post=post).order_by("-published")
+    is_liked = False
+    if request.user.is_authenticated:
+        like_key = "post_like_{post_id}"
+        is_liked = request.session.get(like_key, False)
+
+    context = {"id": post.id, "title": post.title, "author": post.author, "description": post.description, "content": post.content, "is_post_liked": is_liked,"post_comments":post_comments}
+
+    return render(request, "quickcomm/post.html", context)
+
+@login_required
+def post_like(request, post_id):
+    objects = Post.objects.all()
+    post = objects.get(pk=post_id)
+
+    if request.user.is_authenticated:
+        print("comes in here?")
+        author = Author.objects.all().get(user=request.user)
+        like_obj, created_obj = Like.objects.get_or_create(post=post, author=author)
+        if not created_obj:
+            like_obj.delete()
+        like_key = f"post_like_{post_id}"
+        request.session[like_key] = not created_obj
+
+    return redirect("post_view", post_id=post_id)
+
+@login_required
+def post_comment(request, post_id):
+    objects = Post.objects.all()
+    post = objects.get(pk=post_id)
+    if request.method == 'POST':
+        form = CreateCommentForm(request.POST)
+        if form.is_valid():
+            author = Author.objects.all().get(user=request.user)
+            comment = Comment()
+            comment.post = post
+            comment.author = author
+            comment.comment = form.cleaned_data['comment']
+            comment.save()
+            return HttpResponseRedirect(reverse('post_comment', args=[post.pk]))
+        
+    else:
+        form = CreateCommentForm()
```

Comment:
Remove these two lines, as they do not do anything. This new form doesn't go anywhere. 

Also the way the code is written, there will be no response if there is an error, but it's okay for now.